### PR TITLE
Improves types for a few variables APIs

### PR DESCRIPTION
**Pull request by connorjsmith**
Improves types for
* resolvedVariableModes
* explicitVariableModes
* valuesByMode

### CODE REVIEW

File: plugin-api.d.ts [hunk 1]
```diff
@@ -1600,7 +1604,6 @@ interface StickyNode extends OpaqueNodeMixin, MinimalFillsMixin, MinimalBlendMix
   readonly text: TextSublayerNode
   authorVisible: boolean
   authorName: string
-  isWideWidth: boolean
```
Comment (jefflee-figma) [inline plugin-api.d.ts:1603]: Sorry for the drive-by review, but I recently added this property. Was removing it an intentional change?

**Review by ymichael — APPROVED** [review]
lg - pending jeff's comment!
